### ISSUE TITLE
A11y: Windows High Contrast Mode - Home page - gun violence graph

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/clean-and-green-philly#747]
## Describe the bug
1. When in Windows High Contrast Mode / dark the labels on the gun violence graph are hard to see. 
Things that are not visible:
- Title - Annual Shootings in Philadelphia 2015 - 2023
- The x & y axis labels
- The "Fatal" and "non-fatal" text labels
- "Source: Philadelphia Police Department"

Normal
<img width="498" alt="Screenshot 2024-07-15 at 10 27 10 AM" src="https://github.com/user-attachments/assets/f059aeee-c4ca-4a0e-8352-27eb0897cf84">

High Contrast Mode
<img width="1002" alt="Screenshot 2024-07-15 at 10 26 34 AM" src="https://github.com/user-attachments/assets/8d0f435d-bae8-4fca-86e5-767cba4f8bce">

You can fix this either by fixing the contrast of the text against the background, or you can set the background of the image to always be white. If you choose to go the first route (ideal), check out this article on WHCM special keywords (`Canvas` for background, `CanvasText` for labels, etc.) https://www.smashingmagazine.com/2022/03/windows-high-contrast-colors-mode-css-custom-properties/

## Acceptance Criteria
Open the page in WHCM dark or simulate it using Devtools(see link below)
- [ ] The labels will be visible against the background.

## Additional context
* https://www.smashingmagazine.com/2022/06/guide-windows-high-contrast-mode/
* How to simulate WHCM in Chrome Devtools - https://devtoolstips.org/tips/en/emulate-forced-colors/
*  https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/

**Comment by zheyichn** [comment]
The gun crime graph is a static PNG file named `graphic-guncrimes.png` in the `images` folder where color preferences cannot be manually applied. Should we create a component for this graph instead?

**Comment by bacitracin** [comment]
@zheyichn Is there a way to set the background color of the PNG? Or add one to the container that holds it? That could be a quick fix.

Your suggestion of switching it to a component would allow customizability for text zoom and WHCM system color changes, but I'm not sure if it's worth the effort.

**Comment by zheyichn** [comment]
Hi @bacitracin, may I be assigned this issue?

Since the text is within a PNG image, the first route doesn't work. I will set the backgroud of the image to be white to fix the issue.

**Comment by bacitracin** [comment]
@zheyichn Assigned! Thanks for looking into this. And yes, I think it's time to break out our CSS, good suggestion.
